extragear/kmymoney/blacklist_mac.txt

```diff
@@ -1,2 +1,3 @@
 .*PlugIns/kf5/org\.kde\.kwindowsystem\.platforms.*
 .*PlugIns/org\.kde\.kdecoration2.*
+lib/grantlee/.*
```

Comment:
why?

Reply:
As discussed, codesign refuses to sign it. After researching it further it is due to the dot in grantlee/5.2 path. Also explains why it doesn't take the tcl8.6 in.

Filed a bug report with craft and closing this pull request for now: https://bugs.kde.org/show_bug.cgi?id=419272